install.sh

```diff
@@ -35,4 +36,25 @@ do
     fi
 
     chmod +x "$dest"
-done
\ No newline at end of file
+done
+
+# Install git-secrets command
+function command_exists {
+  #this should be a very portable way of checking if something is on the path
+  #usage: "if command_exists foo; then echo it exists; fi"
+  type "$1" &> /dev/null
+}
+
+if [ ! -f /usr/local/bin/git-secrets ]; then 
+    echo "You don't have git secrets installed. It will be used by some hooks."
+    read -p "Would you like to install git secrets (y/n)? " -n 1 -r
+    echo   
+    if [[ $REPLY =~ ^[Yy]$ ]]; then
+        echo "Installing git-secrets"
+        download_dir="/tmp/git-secrets-$(date +%s)"
+        git clone --depth 1 https://github.com/awslabs/git-secrets.git $download_dir
```

Comment:
@pp-ps alternatively we can use this fork for Azure: https://github.com/msalemcode/git-secrets
I don't actually have an opinion

Reply:
I've removed this part, maybe it's better if we leave to the right platform (Brew, git, package...) the setup.